heat/core/rounding.py

```diff
@@ -328,6 +340,90 @@ def round(
 DNDarray.round.__doc__ = round.__doc__
 
 
+def sign(x: DNDarray, out: Optional[DNDarray] = None) -> DNDarray:
+    """
+    Returns an indication of the sign of a number, element-wise. The definition for complex values is equivalent to :math:`x / \\sqrt{x \\cdot x}`.
+
```

Comment:
Mention "equivalent to numpy.sign" somewhere in the docs?

Reply:
I don't think that it is really necessary because I do mention the definition for complex values. Also, PyTorch's sign function does not work for complex values.
We can add it, though, if you want to put strong emphasis on that.